### PR TITLE
fix: use caddy remote module (instead of local one)

### PR DESCRIPTION
**Pull request by rpoisel**
The referenced tag does not exist (anymore). This should fix this. Thanks!

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.18
 replace github.com/caddyserver/caddy/v2 => ../caddyserver/caddy
```
Comment (mohammed90) [inline go.mod:5]: This is the fix

```suggestion
```

**Review by mohammed90 — CHANGES_REQUESTED** [review]
Thank you for the PR, but the referenced commit actually exists. The `replace` directive is not right to stay there.

File: go.mod (hunk 1 above)
Comment (rpoisel) [reply go.mod]: Thanks for the hint and your quick response. I added two commits which in total only change what you requested.

Comment (mohammed90) [reply go.mod]: Great. Can you please run `go mod tidy`? There are changes to `go.sum` to be applied by the `go mod tidy` run

**Review by mholt — APPROVED** [review]
Huh, oops. I can't remember how this slipped in there, but there we have it. Thanks!

**Comment by rpoisel** [comment]
Thanks for your help and your patience :-) I also "fixed" the subject of this PR.

**Review by mohammed90 — APPROVED** [review]
Perfect! Thank you so much

**Comment by mholt** [comment]
Thank you both :D

**Pull request merged**